### ISSUE TITLE
ThreeJs Facemesh

### ISSUE THREAD

**Issue by mayhemantt** [hiukim/mind-ar-js#439]
I am trying to use the facemesh in the three (face-tracking) for occlusion, for that I need to push the face mesh backward as to be close to the face.
I tried facemesh.position.set(0,0,-0.3) 
to make it go backward in the z, but this had no result how can i achieve the result

**Comment by hiukim** [comment]
right, because the library will control the position of the facemesh object created.

You could clone the facemesh and control the position yourself I suppose?  just update it's pose according to the original mesh with the offset you want.
